lib-es5/v2/search.js

```diff
@@ -111,6 +138,37 @@ var Search = function () {
       options = options || {};
       return api.search(this.to_query(), options, callback);
     }
+  }, {
+    key: 'to_url',
+    value: function to_url(ttl, next_cursor) {
+      var options = arguments.length > 2 && arguments[2] !== undefined ? arguments[2] : {};
+
+      var apiSecret = 'api_secret' in options ? options.api_secret : config().api_secret;
+      if (!apiSecret) {
+        throw new Error('Must supply api_secret');
+      }
+
+      var signingAlgorithm = options.signature_algorithm || config().signature_algorithm || DEFAULT_SIGNATURE_ALGORITHM;
```

Comment:
Search URL can be only signed with SHA256

Reply:
Changed it to `sha256` always